### ISSUE TITLE
Put link on description too

### ISSUE THREAD

**Issue by lacivert** [fraywing/textAngular#1474]
I think the link  http://www.textangular.com  should be in description too

**Comment by pbassut** [comment]
You mean inside `package.json`? If so, why?

**Comment by lacivert** [comment]
Nope, on the github page https://github.com/textAngular/textAngular, on top next  to the description

**Comment by lacivert** [comment]
To save time for the newcomers. I know there is a link on the top of readme file but I think that's far more better to see the demo in the first sight link on the top of github page. Otherwise, some more seconds to get the website to see demos
